src/components/TextArea/TextArea.css

```diff
@@ -0,0 +1,136 @@
+@import "../../styles/theme/index.css";
+
+
+:root {
+  --textarea-background-color: var(--structure-color-100);
+
+  --textarea-large-border-radius: var(--round-m);
+  --textarea-medium-border-radius: var(--round-m);
+
+  --textarea-large-font-size: var(--paragraph-l);
+  --textarea-large-font-weight: var(--paragraph-font-weight-bold);
+  --textarea-large-line-height: var(--paragraph-xl-line-height);
+
+  --textarea-medium-font-size: var(--paragraph-m);
+  --textarea-medium-font-weight: var(--paragraph-font-weight-bold);
+  --textarea-medium-line-height: var(--paragraph-m-line-height);
+
+  --textarea-large-height: var(--height-xxl);
+  --textarea-medium-height: var(--height-l);
+
+  --textarea-large-padding: var(--spacing-vertical-s) var(--spacing-horizontal-m);
+  --textarea-medium-padding: var(--spacing-vertical-s) var(--spacing-horizontal-m) var(--spacing-horizontal-s) var(--spacing-vertical-s);
+
+  --textarea-border: 1px solid var(--structure-color-300);
+  --textarea-border-active: 1px solid var(--primary-color-500);
+  --textarea-border-hover: 1px solid var(--primary-color-300);
+  --textarea-border-negative: 1px solid var(--error-color);
+  --textarea-border-positive: 1px solid var(--success-color);
+
+  --textarea-color: var(--structure-color-900);
+
+  --textarea-outline-color-focus: var(--primary-color-500);
+
+  --textarea-placeholder-color: var(--structure-color-700);
+  --textarea-placeholder-font-weigth: var(--paragraph-font-weight-normal);
+
+  --textarea-label-font-size: var(--paragraph-l);
+  --textarea-label-font-weight: var(--paragraph-font-weight-normal);
+  --textarea-label-line-height: var(--paragraph-l-line-height);
+  --textarea-label-height: var(--height-s);
+  --textarea-label-color: var(--structure-color-800);
+
+  --textarea-span-font-size: var(--paragraph-m);
+  --textarea-span-font-weight: var(--paragraph-font-weight-normal);
+  --textarea-span-line-height: var(--paragraph-m-line-height);
+  --textarea-span-height: var(--height-xs);
+  --textarea-span-color: var(--structure-color-700);
+
+  --textarea-message-negative-color: var(--error-color);
+  --textarea-message-positive-color: var(--success-color);
+}
+
+.textarea {
+  overflow: auto;
```

Comment:
could we remove this property? it's better if the textarea's container handles the position